### PR TITLE
[query/azlogs] update docs and prep for GA

### PR DESCRIPTION
**Pull request by gracewilcox**
* Update readme
* Update changelog
* Prep for GA

### CODE REVIEW

File: sdk/monitor/query/azlogs/README.md [hunk 1]
```diff
@@ -96,19 +98,28 @@ the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_conta
...
 [kusto_query_language]: https://learn.microsoft.com/azure/data-explorer/kusto/query/
 [kusto_to_sql]: https://learn.microsoft.com/azure/data-explorer/kusto/query/sqlcheatsheet
 [log_analytics_workspace]: https://learn.microsoft.com/azure/azure-monitor/logs/log-analytics-workspace-overview
 [log_analytics_workspace_create]: https://learn.microsoft.com/azure/azure-monitor/logs/quick-create-workspace
 [logs_overview]: https://learn.microsoft.com/azure/azure-monitor/logs/data-platform-logs
+[metrics_overview]: https://learn.microsoft.com/azure/azure-monitor/essentials/data-platform-metrics
 [monitor_docs]: https://learn.microsoft.com/azure/azure-monitor/
 [monitor_rest_docs]: https://learn.microsoft.com/rest/api/monitor/
+[pkg_go]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/query/azlogs
+[samples]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/query/azlogs#pkg-examples
```
Comment (chlowell) [inline sdk/monitor/query/azlogs/README.md:120]:
```suggestion
```

**Review by chlowell — APPROVED** [review]

File: sdk/monitor/query/azlogs/README.md (hunk 1 above)
Comment (gracewilcox) [reply sdk/monitor/query/azlogs/README.md]: fixed in [83967e6](https://github.com/Azure/azure-sdk-for-go/pull/22727/commits/83967e64074f79c0659673582b0f4e5a5bbc8b88)

**Pull request merged**